Dialogs/Dialogs.pt-BR.json

```diff
@@ -0,0 +1,47 @@
+{
+  "AscendingRpmValues": "Valores de RPM ascendentes",
+  "CalibrationCommand": "Comando (%)",
+  "CalibrationControlCardInstructions": "Use o seguinte cartão para ajustar a % do ventilador e observar o RPM.",
+  "CalibrationPointPercent": "% do ponto",
+  "CalibrationSpeed": "Velocidade (RPM)",
+  "CalibrationTable": "Tabela de Calibração",
+  "ControlHelpPart1": "A seção \"Controles\" mostra todas os cabeçalhos e ventoinhas disponíveis que foram detectados.",
+  "ControlHelpPart2": "Você pode atribuir uma curva de ventoinha ou entrar no controle manual a partir do menu de três pontos de cada cartão.",
+  "ControlHelpPart3": "No menu de três pontos de cada cartão, você pode \"emparelhar\" um sensor de velocidade do ventilador da seção abaixo, que exibirá o RPM no cartão de controle.",
+  "ControlHelpPart4": "As seguintes opções avançadas estão disponíveis ao clicar na parte inferior de um cartão:",
+  "CurveHelpPart1": "A seção \"Curvas\" mostra todas as curvas que foram criadas.",
+  "CurveHelpPart2": "Para criar uma nova curva, clique no grande botão de \"+\" no canto inferior direito e selecione um dos tipos disponíveis.",
```

Comment:
```suggestion
  "CurveHelpPart2": "Para criar uma nova curva, clique no botão grande \"+\" no canto inferior direito e selecione um dos tipos disponíveis.",
```

Reply:
"botão grade" is not necessary, I will aplly just "botão"